templates/data/generic_external_module_data.yaml

```diff
@@ -54,7 +54,7 @@ templates:
                     MODULE_REGISTRY: "europe-west4-docker.pkg.dev/sap-se-cx-kyma-wookiee/dev-registry/unsigned" 
                     MODULE_VERSION: "0.0.${PULL_NUMBER}"
                     IMG: "eu.gcr.io/kyma-project/keda-manager:PR-${PULL_NUMBER}"
-                  run_if_changed: "^operator/(go.mod|go.sum)$|^operator/*/(.*.go|Makefile|.*.sh)"
+                  run_if_changed: "^operator/(go.mod|go.sum)$|^operator/*/(.*.go|Makefile|.*.sh)|^operator/PROJECT$|^operator/config/*/.*$"
```

Comment:
Why there is a need to rebuild the module when shell scripts have changed? Maybe the module should be rebuilt on any change in the code and yaml on entire operator directory?

Reply:
- we are developing `keda-manager` as early adopters (it's the 1st module to be created)
- we most likely will update the job sooner or later (there is/will be an issue for flattening the project) ATM those conditions are good enough for us